src/stream-analytics/azext_stream_analytics/_help.py

```diff
@@ -0,0 +1,370 @@
+               --events-late-arrival-max-delay-in-seconds "13"
+"""
+
+helps['stream-analytics job delete'] = """
+    type: command
+    short-summary: Deletes a streaming job.
+    examples:
+      - name: Delete a streaming job
+        text: |-
+               az stream-analytics job delete --resource-group "sjrg6936" --name "sj59"
+"""
+
+helps['stream-analytics job show'] = """
+    type: command
+    short-summary: Gets details about the specified streaming job.
+    examples:
+      - name: Get a streaming job
+        text: |-
+               az stream-analytics streaming-job show --resource-group "sjrg6936" --name "sj59"
+      - name: Get a streaming job and expand its inputs, outputs, transformation, and functions
+        text: |-
+               az stream-analytics streaming-job show --resource-group "sjrg3276" --name "sj7804" --expand
+"""
+
+helps['stream-analytics job list'] = """
+    type: command
+    short-summary: Lists all of the streaming jobs in the specified resource group.
+    examples:
+      - name: List all streaming jobs
+        text: |-
+               az stream-analytics job list --resource-group "sjrg3276"
+      - name: List all streaming jobs and expand their inputs, outputs, transformation, and functions
+        text: |-
+               az stream-analytics job list --resource-group "sjrg6936" --expand
+"""
+
+helps['stream-analytics job start'] = """
+    type: command
+    short-summary: Starts a streaming job.
+    examples:
+      - name: Start a streaming job with LastOutputEventTime output start mode
+        text: |-
+               az stream-analytics job start --resource-group "sjrg6936" --name "sj59" --output-start-mode LastOutputEventTime
+      - name: Start a streaming job with JobStartTime output start mode
+        text: |-
+               az stream-analytics job start --resource-group "sjrg6936" --name "sj59" --output-start-mode JobStartTime
+      - name: Start a streaming job with CustomTime output start mode
+        text: |-
+               az stream-analytics streaming-job start --resource-group "sjrg6936" --name "sj59" --output-start-mode CustomTime --output-start-time 2020-01-01T00:00:00Z
+"""
+
+helps['stream-analytics job stop'] = """
+    type: command
+    short-summary: Stops a running streaming job.
+    examples:
+      - name: Stop a streaming job
+        text: |-
+               az stream-analytics job stop --resource-group "sjrg6936" --name "sj59"
+"""
+
+helps['stream-analytics input'] = """
+    type: group
+    short-summary: Commands to manage stream-analytics input.
+"""
+
+helps['stream-analytics input create'] = """
+    type: command
+    short-summary: Creates an input or replaces an already existing input under an existing streaming job.
+    examples:
+      - name: Create an input
+        text: |-
+               az stream-analytics input create --resource-group "sjrg8440" --job-name "sj9597" --name \\
```

Comment:
is the serialization necessary? can we define default deliminitor?

Reply:
there are several different types of serialization(json, csv, avro, others), each one may have different properties.

Reply:
ok

Reply:
can we provide examples for json, avro, others too?